thinkbox-deadline/azuredeploy.json

```diff
@@ -0,0 +1,404 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+
+    "parameters": {
+
+        "location": 
+        {
+            "type": "string",
+            "allowedValues": 
+            [
+                "West US",
+                "East US",
+                "West Europe",
+                "East Asia",
+                "Southeast Asia"
+            ],
+            "metadata": 
+            {
+                "description": "The region to deploy the resources into"
+            }
+        },
+
+        "publicDnsName": 
+        {
+            "type": "string",
+            "metadata": 
+            {
+                "description": "Unique public DNS prefix for the deployment. The fqdn will look something like '<dnsname>.westus.cloudapp.azure.com'. Up to 62 chars, digits or dashes, lowercase, should start with a letter: must conform to '^[a-z][a-z0-9-]{1,61}[a-z0-9]$'."
+            }
+        },
+
+        "storageAccount": 
+        {
+            "type": "string",
+            "metadata": 
+            {
+                "description": "Name of the Storage Account. This has to be a uinique name, up to 24 chars, all lowercase."
+            }
+        },
+
+        "adminUsername": 
+        {
+            "type": "string",
+            "metadata": 
+            {
+                "description": "The name of the administrator of the new VM. Exclusion list: 'admin','administrator'"
+            }
+        },
+        "adminPassword": 
+        {
+            "type": "securestring",
+            "metadata": 
+            {
+                "description": "The password for the administrator account of the new VM"
+            }
+        },
+
+        "numberOfSlaves":
+        {
+            "type": "int",
+            "defaultValue": 2,
+            "metadata": 
+            {
+                "description": "The number of slave vms to start."
+            }
+        }
+    },
+
+    "variables": {
+        "vnetAddressRange": "10.0.0.0/16",
+        "subnetAddressRange": "10.0.0.0/24",
+        "subnetName":  "Subnet",
+        "subnet-id": "[concat(resourceId('Microsoft.Network/virtualNetworks','VNET'),'/subnets/',variables('subnetName'))]"
+    },
+
+    "resources": [
+
+        {
+            "apiVersion": "2015-05-01-preview",

```

Comment:
Consider making apiVersion a variable so that it becomes easy to move to new versions